infrastructure/quick-deploy/aws/armonik/variables.tf

```diff
@@ -54,14 +54,20 @@ variable "monitoring" {
   default     = {}
 }
 
-# Polling delay to MongoDB
-# according to the size of the task and/or the application
-variable "mongodb_polling_delay" {
-  description = "Polling delay to MongoDB according to the size of the task and/or the application"
-  type = object({
-    min_polling_delay = string
-    max_polling_delay = string
-  })
+
+# Extra configuration
+variable "extra_conf" {
+  description = "Add extra configuration in the configmaps"
+  #type = object({
+  #  compute = optional(map(string), {})
+  #  control = optional(map(string), {})
+  #  core    = optional(map(string), {})
```

Comment:
Is it intentional ?

Reply:
It is intentional: only the all-in-one deployments require Terraform 1.3, and can use optionals. The comment here is to document the format expected to "simulate" optionals in previous versions.